package.json

```diff
@@ -51,7 +51,8 @@
     "onFileSystem:gist",
     "onView:gistpad.gists",
     "onView:gistpad.gists.explorer",
-    "onUri"
+    "onUri",
+    "onLanguage:markdown"
```

Comment:
If we limit this to only gist files (which I think we should), then I don't believe we'd need this activation event anymore.